src/gui/DatabaseSettingsWidget.cpp

```diff
@@ -75,6 +78,31 @@ void DatabaseSettingsWidget::load(Database* db)
         m_ui->historyMaxSizeCheckBox->setChecked(false);
     }
 
+    // FIXME don't hardcode this here
+    m_ui->cipherCombo->addItem("AES-256", KeePass2::CIPHER_AES.toByteArray());
```

Comment:
You cannot simply add this here without clearing the combo box first. Otherwise you have duplicate entries when reopening the dialog. The combobox lifetime is not necessarily bound to this widget.